packages/ffe-datepicker-react/src/calendar/Header.js

```diff
@@ -1,50 +1,139 @@
+            >
+                <div className="ffe-calendar__header-wrapper">
+                    <div className="ffe-calendar__header-define" />
+                    <div className="ffe-calendar__header-break ffe-calendar__header-break-1" />
+                    <div className="ffe-calendar__header-block" />
+                    <div className="ffe-calendar__header-define" />
+                    <div className="ffe-calendar__header-break ffe-calendar__header-break-2" />
+                    <div className="ffe-calendar__header-block" />
+
+                    <div
+                        id={`${datepickerId}-view1`}
+                        className="ffe-calendar__header-content"
+                    >
+                        <div className="ffe-calendar__header-inner-1">
+                            <div>
+                                <Dropdown
+                                    value={year}
+                                    tabIndex="-1"
+                                    onChange={selectYearHandler}
+                                >
+                                    {renderYearOptions()}
+                                </Dropdown>
+                            </div>
+                            <div>
+                                <MonthPicker
+                                    previousMonthHandler={previousMonthHandler}
+                                    nextMonthHandler={nextMonthHandler}
+                                    month={month}
+                                />
+                            </div>
+                        </div>
+                    </div>
+                    <div
+                        id={`${datepickerId}-view2`}
+                        className="ffe-calendar__header-content"
+                    >
+                        <div className="ffe-calendar__header-inner-2">
+                            <div>
+                                <Dropdown
+                                    value={year}
+                                    tabIndex="-1"
+                                    onChange={selectYearHandler}
+                                >
+                                    {renderYearOptions()}
+                                </Dropdown>
+                            </div>
+                            <div>
+                                <MonthPicker
+                                    previousMonthHandler={previousMonthHandler}
+                                    nextMonthHandler={nextMonthHandler}
+                                    month={month}
+                                />
+                            </div>
+                            <div>
+                                <SecondaryButton onClick={todayHandler}>
+                                    {todayLabel}
+                                </SecondaryButton>
+                            </div>
+                        </div>
+                    </div>
+                    <div
```

Comment:
Vad er det som skjer her? Har vi samma markup 3 ganger som for olika devicer? Det kan vi ikke ha pga av screen readers.

Reply:
[Artificial element breakpoints](https://github.com/johann1301h/almostAlwaysBounce). Yes, it is necessary using this technique. I did not think about screen readers. I suggest using [css-element-queries](https://github.com/marcj/css-element-queries) instead if this design with breakpoints is to be implemented.

Reply:
Its a creative solution so you get an A for effort.  :)

I think even if you could live with maintaining 3 sets of markups. The Accessibility aspect is what really makes it fall short.